### PR TITLE
Remove HIV bulk upload link from admin page

### LINKED ISSUE #7951: Clean up HIV bulk upload beta from support admin dashboard
### Description

The bulk upload beta link is now visible on the support admin dashboard in production. We were originally keeping this around in case we decided to support HIV bulk upload via a separate schema (see #7078 and #7228). Now that it's supported in our regular bulk upload, we should clean up this code.

### Proposed solution

Clean up the bulk upload beta code, including the link on support admin dashboard, upload pages, and `/upload/hiv-results` endpoint.

### Additional context
Screenshot 2024-07-25 at 1 49 55 PM

### PR DESCRIPTION
**Pull request by bobbywells52**
# Backend/FRONTEND PULL REQUEST

## Related Issue
Resolves #7951 

## Changes Proposed

- Remove the `/upload/hiv-results` endpoint 
- Remove the beta link to the HIV upload form on the admin support page
- Remove the corresponding HIV upload form 
- Delete or refactor obsolete tests

## Additional Information
- As it was outside the scope of the ticket, I did not remove the `bulkHivUploadEnabled` flag from the code base but my work on this ticket makes me wonder if we need the flag at all anymore.

## Testing
Live for review on dev6, reviewer should confirm the following:
- that`/admin/hiv-csv-upload` is no longer a valid route.
- that the link to the HIV Bulk Upload Beta is no longer on the admin support page
- that you can no longer hit the `/upload/hiv-results` endpoint

### CODE REVIEW

File: backend/src/main/java/gov/cdc/usds/simplereport/api/uploads/FileUploadController.java [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 import static gov.cdc.usds.simplereport.api.Translators.parseUUID;
 import static gov.cdc.usds.simplereport.config.WebConfiguration.CONDITION_AGNOSTIC_RESULT_UPLOAD;
-import static gov.cdc.usds.simplereport.config.WebConfiguration.HIV_RESULT_UPLOAD;
```
Comment (emyl3) [inline backend/src/main/java/gov/cdc/usds/simplereport/api/uploads/FileUploadController.java:5]: Can we remove where we define this as well?

**Review by emyl3 — COMMENTED** [review]
Left a comment about one more thing to remove! Nice cleanup, Bobby! 🧹

File: backend/src/main/java/gov/cdc/usds/simplereport/api/uploads/FileUploadController.java (hunk 1 above)
Comment (bobbywells52) [reply backend/src/main/java/gov/cdc/usds/simplereport/api/uploads/FileUploadController.java:5]: Nice catch -- updated

**Review by fzhao99 — APPROVED** [review]

**Review by emyl3 — APPROVED** [review]
LGTM!

**Pull request merged**